redbot/cogs/audio/audio.py

```diff
@@ -467,7 +469,8 @@ def _restart_connect(self):
         info_msg = _(
             "This setting is only for bot owners to set a localtracks folder location "
             "if the Lavalink.jar is being ran from outside of the Audio data directory.\n"
-            "In the example below, the full path for 'ParentDirectory' must be passed to this command.\n"
+            "In the example below, the full path for "
```

Comment:
Why?

Reply:
This is due to line length being longer than 99 characters, one bellow it’s the same but the backlash bellow is so that when this its printed to a user it isn’t forced into a new line

Reply:
Didn't notice the line length on this one was that long, mildly surprised black didn't complain about it before given that it was already inside parens.